udata/templates/search.html

```diff
@@ -89,7 +89,7 @@
     created=(_('Creation date'), 'desc'),
     last_modified=(_('Last modification date'), 'desc'),
     datasets=(_('Datasets used'), 'desc'),
-    followers=(_('Stars'), 'desc'),
+    followers=(_('Stars'), 'desc')
```

Comment:
why remove the trailing commas? make a clean diff when you add a line, we have some pretty much everywhere

Reply:
My bad It's a test change that I forgot to remove. Fixed